apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -3,8 +3,11 @@ import {
   SET_SCRIPT,
   getSelectedScriptName
 } from '@cdo/apps/redux/scriptSelectionRedux';
-import i18n from '@cdo/locale';
 import experiments from '@cdo/apps/util/experiments';
+import {
+  inProgressFakeTimestamp,
+  notStartedFakeTimestamp
+} from '@cdo/apps/templates/sectionAssessments/assessmentsTestHelpers';
```

Comment:
i would actually invert this dependency and have the reducer define our timestamp constants, and our `assessmentsTestHelpers` import them

Reply:
✅